templates/waf_web_acl.cfn.yml

```diff
@@ -21,8 +21,19 @@ Resources:
         MetricName: "WafWebACL"
         SampledRequestsEnabled: true
       Rules:
-        - Name: !Sub "${Environment}-${Platform}-XSSRule"
+        - Name: !Sub "${Environment}-${Platform}-RateLimitRule"
           Priority: 1
+          Action:
+            Block: {}
+          Statement:
+            RateBasedStatement:
```

Comment:
add EvaluationWindowSec with 300, yes its default but best to state it. 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-webacl-ratebasedstatement.html#cfn-wafv2-webacl-ratebasedstatement-evaluationwindowsec